src/util/validators.ts

```diff
@@ -60,3 +60,11 @@ export function validateEmailField(value: string): string {
 
   return '';
 }
+
+export function validateCaptcha(value: string | null | undefined): string {
+  const message = i18n.t(
```

Comment:
please inline this message in the return value. this way we will call the translation function only when it is actually needed, instead of calling it each time the `validateCaptcha` function is called

a small performance optimization, but they stack up over time